x-pack/plugins/synthetics/server/synthetics_service/formatters/public_formatters/common.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { arrayToJsonFormatter, secondsToCronFormatter } from '../formatting_utils';
+import { arrayFormatter, stringToObjectFormatter } from './formatting_utils';
```

Comment:
This is why I asked to have the private formatters  (arrayToJsonFormatter) moved into the private folder. It's confusing with the two same imports which is which.

Reply:
Also, pretty sure we shouldn't be importing the top one at all, perhaps with the exception of `secondsToCronFormatter`.

If `secondsToCron` is common between private and public, could you move it to a common section?